PHP distribtest broken: distribtest.php_linux_x64_jessie

The distribtest php_linux_x64_jessie is broken consistently
```
libtool: compile:  g++ -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN -D_HAS_EXCEPTIONS=0 -DNOMINMAX -DGRPC_ARES=0 -I. -I/tmp/pear/temp/grpc -DPHP_ATOM_INC -I/tmp/pear/temp/pear-build-defaultuser6dy3Qf/grpc-1.14.0dev/include -I/tmp/pear/temp/pear-build-defaultuser6dy3Qf/grpc-1.14.0dev/main -I/tmp/pear/temp/grpc -I/usr/include/php5 -I/usr/include/php5/main -I/usr/include/php5/TSRM -I/usr/include/php5/Zend -I/usr/include/php5/ext -I/usr/include/php5/ext/date/lib -I/tmp/pear/temp/grpc/include -I/tmp/pear/temp/grpc/src/php/ext/grpc -I/tmp/pear/temp/grpc/third_party/boringssl/include -I/tmp/pear/temp/grpc/third_party/address_sorting/include -DHAVE_CONFIG_H -std=c++11 -fno-exceptions -fno-rtti -g -O2 -D PB_FIELD_16BIT=1 -c /tmp/pear/temp/grpc/src/core/lib/security/credentials/alts/alts_credentials.cc  -fPIC -DPIC -o src/core/lib/security/credentials/alts/.libs/alts_credentials.o
In file included from /tmp/pear/temp/grpc/src/core/lib/security/credentials/alts/grpc_alts_credentials_options.h:26:0,
                 from /tmp/pear/temp/grpc/src/core/lib/security/credentials/alts/alts_credentials.h:26,
                 from /tmp/pear/temp/grpc/src/core/lib/security/credentials/alts/alts_credentials.cc:21:
/tmp/pear/temp/grpc/src/core/tsi/alts/handshaker/transport_security_common_api.h:24:23: fatal error: pb_decode.h: No such file or directory
 #include "pb_decode.h"
                       ^
compilation terminated.
Makefile:644: recipe for target 'src/core/lib/security/credentials/alts/alts_credentials.lo' failed
make: *** [src/core/lib/security/credentials/alts/alts_credentials.lo] Error 1
ERROR: `make' failed
```

https://source.cloud.google.com/results/invocations/787f7c10-1a5e-4b2d-b5ba-8493fa545ea6/targets/github%2Fgrpc/tests

--- a/config.m4
+++ b/config.m4
@@ -9,11 +9,12 @@ if test "$PHP_GRPC" != "no"; then
   PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/src/php/ext/grpc)
   PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/third_party/boringssl/include)
   PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/third_party/address_sorting/include)
+  PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/third_party/nanopb)
 
   LIBS="-lpthread $LIBS"
 
-  CFLAGS="-Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -g -O2 -D PB_FIELD_16BIT=1"
-  CXXFLAGS="-std=c++11 -fno-exceptions -fno-rtti -g -O2 -D PB_FIELD_16BIT=1"
+  CFLAGS="-Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -g -O2 -D PB_FIELD_32BIT=1"
+  CXXFLAGS="-std=c++11 -fno-exceptions -fno-rtti -g -O2 -D PB_FIELD_32BIT=1"
   GRPC_SHARED_LIBADD="-lpthread $GRPC_SHARED_LIBADD"
   PHP_REQUIRE_CXX()
   PHP_ADD_LIBRARY(pthread)

--- a/config.w32
+++ b/config.w32
@@ -638,13 +638,14 @@ if (PHP_GRPC != "no") {
   EXTENSION("grpc", grpc_source, null,
     "/DOPENSSL_NO_ASM /D_GNU_SOURCE /DWIN32_LEAN_AND_MEAN "+
     "/D_HAS_EXCEPTIONS=0 /DNOMINMAX /DGRPC_ARES=0 /D_WIN32_WINNT=0x600 "+
-    "/DPB_FIELD_16BIT "+
+    "/DPB_FIELD_32BIT "+
     "/I"+configure_module_dirname+" "+
     "/I"+configure_module_dirname+"\\include "+
     "/I"+configure_module_dirname+"\\src\\php\\ext\\grpc "+
     "/I"+configure_module_dirname+"\\third_party\\boringssl\\include "+
     "/I"+configure_module_dirname+"\\third_party\\zlib "+
-    "/I"+configure_module_dirname+"\\third_party\\address_sorting\\include");
+    "/I"+configure_module_dirname+"\\third_party\\address_sorting\\include "+
+    "/I"+configure_module_dirname+"\\third_party\\nanopb");
 
   base_dir = get_define('BUILD_DIR');
   FSO.CreateFolder(base_dir+"\\ext");

--- a/templates/config.m4.template
+++ b/templates/config.m4.template
@@ -11,11 +11,12 @@
     PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/src/php/ext/grpc)
     PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/third_party/boringssl/include)
     PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/third_party/address_sorting/include)
+    PHP_ADD_INCLUDE(PHP_EXT_SRCDIR()/third_party/nanopb)
 
     LIBS="-lpthread $LIBS"
 
-    CFLAGS="-Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -g -O2 -D PB_FIELD_16BIT=1"
-    CXXFLAGS="-std=c++11 -fno-exceptions -fno-rtti -g -O2 -D PB_FIELD_16BIT=1"
+    CFLAGS="-Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -g -O2 -D PB_FIELD_32BIT=1"
+    CXXFLAGS="-std=c++11 -fno-exceptions -fno-rtti -g -O2 -D PB_FIELD_32BIT=1"
     GRPC_SHARED_LIBADD="-lpthread $GRPC_SHARED_LIBADD"
     PHP_REQUIRE_CXX()
     PHP_ADD_LIBRARY(pthread)

--- a/templates/config.w32.template
+++ b/templates/config.w32.template
@@ -23,13 +23,14 @@
     EXTENSION("grpc", grpc_source, null,
       "/DOPENSSL_NO_ASM /D_GNU_SOURCE /DWIN32_LEAN_AND_MEAN "+
       "/D_HAS_EXCEPTIONS=0 /DNOMINMAX /DGRPC_ARES=0 /D_WIN32_WINNT=0x600 "+
-      "/DPB_FIELD_16BIT "+
+      "/DPB_FIELD_32BIT "+
       "/I"+configure_module_dirname+" "+
       "/I"+configure_module_dirname+"\\include "+
       "/I"+configure_module_dirname+"\\src\\php\\ext\\grpc "+
       "/I"+configure_module_dirname+"\\third_party\\boringssl\\include "+
       "/I"+configure_module_dirname+"\\third_party\\zlib "+
-      "/I"+configure_module_dirname+"\\third_party\\address_sorting\\include");
+      "/I"+configure_module_dirname+"\\third_party\\address_sorting\\include "+
+      "/I"+configure_module_dirname+"\\third_party\\nanopb");
   <%
     dirs = {}
     for lib in libs:

